db/migrate/20220610094346_add_uniqueness_contrataint_to_library_type_name.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Rubocop want us to add a uniqueness constraint to the database. In practice
+# we're unlikely to reap much form the performance or data-integrity issues,
+# this will be a low volume rarely changing table, but there could be risks
+# of concurrency problems running tasks on deployment.
+class AddUniquenessContrataintToLibraryTypeName < ActiveRecord::Migration[7.0]
```

Comment:
Should this be AddUniquenessConstaintToLibraryTypeName. I know it is only semantics but probably worth changing now?